firmware/font/bdftokff.py

```diff
@@ -31,9 +31,9 @@
 FONT14 = "ter-u14n"
 FONT16 = "ter-u16n"
 FONT24 = "ter-u24b"
-KR_CN_14 = "FusionPixel-14"
-KR_CN_16 = "unifont-16"
-KR_CN_24 = "NotoSansCJK-24"
+ASIA14 = "FusionPixel-14"
```

Comment:
not a good idea to rename these away from the locales they cover to a geographic area that is less precise (especially since we may have completely different font-file sets per locale/device).